kiva/agg/src/graphics_context.i

```diff
@@ -907,15 +908,15 @@ class Image(GraphicsContextArray):
         """
         # read the file using PIL
         from PIL import Image as PilImage
-        from kiva.compat import piltostring
+
         pil_img = PilImage.open(file)
 
         # Convert image to a numeric array
         if (pil_img.mode not in ["RGB","RGBA"] or
             (cvar.ALWAYS_32BIT_WORKAROUND_FLAG and pil_img.mode != "RGBA")):
             pil_img = pil_img.convert(mode="RGBA")
         depth = pil_depth_map[pil_img.mode]
-        img = frombuffer(piltostring(pil_img),uint8)
+        img = frombuffer(pil_img.tobytes(), uint8)
```

Comment:
I _think_ we can just convert directly to an array here: `numpy.array(pil_img)`.